packages/republik-crowdfundings/lib/scheduler/payments/README.md

```diff
@@ -0,0 +1,43 @@
+# Postfinance Import
+
+The Postfinance import goes through the follwoing steps:
+
+1. If none are configured, `GOTO 4`.
+2. Download all `camt053.xml` and `camt053.tar.gz` files from all configured Sftp servers.
+3. Write these files to the following db table: `postfinanceImports`.
+4. Read all files where `isImported === false` from the db.
+5. Unpack all `tar` files. These contain one `camt053.xml` file and `.tiff` files. The images are scans of over the counter cash payments (Einzahlungsschein).
+6. Convert images to `.jpeg`.
+7. Parse XML files (c.f. [parseCamt053.u.jest.ts](parseCamt053.u.jest.ts)).
+8. Filter out the following payment entries:
+
+- All `DEBIT` entries.
+- All `CREDIT` entries from `STRIPE`.
+- All `CREDIT` entries from `PAYPAL`.
+- All debit card `CREDIT` entries (Online Postfinance Zahlung).
+
+9. Write payment entries to the follwing db table: `postfinancePayments`.
+10. Run the payment matcher.
+11. Send notifications, if all payments are matched.
```

Comment:
```suggestion
11. Send payment reminders for overdue payments, if all payments are matched.
```

Reply:
Currently not the situation, comes in the next PR. Will update this there.